pixl_dcmd/README.md

```diff
@@ -1,2 +1,9 @@
 # PIXL DICOM de-identifier
 
+pip install -r requirements.txt
+
+pip install -e ../../pixl_core/
+
+pip install -e .
+
+pytest
```

Comment:
Should break up into

- what this module does
- how to setup for development
- how to test